.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```

Comment:
We don't want to push up the changes for the `.idea` directory. It's IntelliJ config files that aren't useful for other people. I would recommend adding the `.idea` directory in the `gitignore` file, so that it doesn't push it up.

Reply:
How would I go about doing this?